tests/integration_tests/test_sim.py

```diff
@@ -102,6 +102,23 @@ def test_gps_read(self, qtbot, sim_app, device_type):
             assert sim_app.rocket_data.last_value_by_device(device_type, DataEntryIds.LONGITUDE) == vals[1]
             assert sim_app.rocket_data.last_value_by_device(device_type, DataEntryIds.GPS_ALTITUDE) == vals[2]
 
+    def test_pin_mode(self, qtbot, sim_app, device_type):
+        test_vals = [
+            (1, 1),
+            (12, 0),
+            (5, 0),
+            (1, 0),
+            (2, 1),
+        ]
+
+        hw = get_hw_sim(sim_app, device_type)
+        for vals in test_vals:
+            hw.set_pin_mode(vals[0], vals[1])
+            flush_packets(sim_app, device_type)
+
+            assert hw.get_pin_mode(vals[0]) == vals[1]
```

Comment:
Currently this is acting as more of a test of hw_sim. However the test_sim.py file is meant more as a test of the FW. Once we have the 3rd pin state representing the default unset state (as per my other comment) we should use this test to check that some pins are being set to something.

Reply:
Updated to check against pins set in FW

484666c